### PR TITLE
Added Omnibox experiment explainer

### PR DESCRIPTION
**Pull request by erdraud**

### CODE REVIEW

File: explainers/Omnibox/explainer.md
Comment (captainbrosset) [inline explainers/Omnibox/explainer.md:8]: This link only works for Microsoft employees. You might want to remove it.

File: explainers/Omnibox/explainer.md
Comment (captainbrosset) [inline explainers/Omnibox/explainer.md:13]:
```suggestion
You’ll notice that many of the commands in this feature can be found in the [Command Menu]( https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/command-menu/) in DevTools, so why have these features in the address bar (from this point forward referred to as the omnibox)? We know that users would like a faster way to complete their tasks in DevTools, and the Omnibox Commands feature enables users to do so without having to first open the Developer Tools. Additionally, we’ve included features that are specific to the browser window: Edge internal urls and last viewed pages.
```

File: explainers/Omnibox/explainer.md
Comment (captainbrosset) [inline explainers/Omnibox/explainer.md:40]:
```suggestion
* Commands shouldn't conflict with search engine searches
```

**Review by captainbrosset — APPROVED** [review]

File: explainers/DocumentationSearch/explainer.md
Comment (erdraud) [inline explainers/DocumentationSearch/explainer.md:4]: @TheLarkInn I used a lot of text from your document. I can keep your name and add your github account or remove your name. Let me know what you prefer.

File: explainers/Omnibox/explainer.md
Comment (erdraud) [reply explainers/Omnibox/explainer.md]: Done!

File: explainers/DocumentationSearch/explainer.md
Comment (TheLarkInn) [reply explainers/DocumentationSearch/explainer.md:4]: Whatever you prefer is fine. I don't mind either :+1:

**Comment by captainbrosset** [comment]
@erdraud do you think we can merge this? Do you need reviews from other people before merging?

**Comment by erdraud** [comment]
@captainbrosset I can't merge it myself, but I this is ready to go.

**Pull request merged**
